features/junit_format.feature

```diff
@@ -538,6 +538,7 @@
       """
     And the file "junit/default.xml" should be a valid document according to "junit.xsd"
 
+  @php-version @php5.3 @php5.4
```

Comment:
@stof not sure if this is what you meant, perhaps just `php5.3` if it works as a "minimum version"